include/pmacc/particles/ParticlesBase.kernel

```diff
@@ -337,8 +344,8 @@ namespace pmacc
          *
          * @warning this kernel can only run with a double checker board
          */
-        template<typename T_ParBox, typename Mapping, typename T_Acc>
-        DINLINE void operator()(T_Acc const& acc, T_ParBox pb, Mapping mapper) const
+        template<typename T_ParBox, typename Mapping, typename T_Acc, typename T_Idx>
```

Comment:
Maybe we should call the type `T_SuperCellIDX`

Reply:
I don't mind it, but also think it's not better (nor worse). Do you want it changed?

Reply:
no not required, was only a suggestion.